README.md

```diff
@@ -75,7 +78,7 @@ scope-az:
         #- source-file-modifiers
         #- source-file-merger
         - source-file-extension
-    output-folder: $(az-output-folder)
+    output-folder: $(output-folder)
 

```

Comment:
feels az-output-folder is the right one to use, isn't it? and az-output-folder is the same as output-folder now, isn't it?